index.html

```diff
@@ -46,8 +46,8 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
   </tr>
 </table>
-
-<h2 hidden>Game over</h2>
+<hr />
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion

<h2 hidden>Game over, refresh to play again</h2>
```

Fixed misunderstanding